install/install_csm_services.md

```diff
@@ -163,6 +163,26 @@ The following error may occur during the `Deploy CSM Applications and Services`
 
 1. Running the `yapl` command again is expected to succeed.
 
-### 5.2 `Setup Nexus` known issues
+### `Error releasing chart` known issues
```

Comment:
Please include this in the table of contents.

Reply:
good catch. I add it to the toc.